packages/inngest/src/helpers/promises.ts

```diff
@@ -35,12 +35,12 @@ export const createFrozenPromise = (): Promise<unknown> => {
  * Returns a Promise that resolves after the current event loop's microtasks
  * have finished, but before the next event loop tick.
  */
-export const resolveAfterPending = (count = 1000): Promise<void> => {
+export const resolveAfterPending = (count = 10): Promise<void> => {
```

Comment:
Only recursively queue 10 microtasks. 1,000, while it benchmarks in well under a millisecond, can get too heavy for its intended use.